src/resources/postcss/rsvp/_actions.pcss

```
@@ -104,6 +104,7 @@
 
 	.tribe-tickets__rsvp-actions-success-going-text {
 		margin-left: var(--spacer-1);
+		tex-transform: capitalize;
```

Was this meant to be capitalized?

```suggestion
		text-transform: capitalize;
```

---

Exactly! Thanks for the heads up. Butterfly keyboard was the worse decision apple ever made.